pkg/thing/interactors/unregister_thing.go

```diff
@@ -0,0 +1,39 @@
+package interactors
+
+import (
+	"errors"
+)
+
+func (i *ThingInteractor) Unregister(authorization, id string) error {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported method ThingInteractor.Unregister should have comment or be unexported](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=5019174)